File: tfh-backend/src/main/java/com/example/tfhbackend/dto/MenuItemDTO.java
package com.example.tfhbackend.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Builder
@Data
public class MenuItemDTO {
    private Long id;
    private String name;
    private Double price;
    private String image;
    private String category;
}

File: tfh-mobile/app/src/main/java/com/example/tfhmobile/dto/MenuItemDTO.kt
package com.example.tfhmobile.dto

class MenuItemDTO constructor(
    private val id: Long,
    private val name: String,
    private val price: Double,
    private val image: String,
    private val category: String
) {
    fun getId(): Long {
        return id
    }

    fun getName(): String {
        return name
    }
}